cmd/osm-bootstrap/osm-bootstrap_test.go

```diff
@@ -1,67 +1,86 @@
 package main
 
 import (
+	"context"
 	"testing"
 
 	tassert "github.com/stretchr/testify/assert"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes"
+	fakeKube "k8s.io/client-go/kubernetes/fake"
+
+	"github.com/openservicemesh/osm/pkg/apis/config/v1alpha1"
+	configClientset "github.com/openservicemesh/osm/pkg/gen/client/config/clientset/versioned"
+	fakeConfig "github.com/openservicemesh/osm/pkg/gen/client/config/clientset/versioned/fake"
 )
 
-func TestCreateDefaultMeshConfig(t *testing.T) {
-	assert := tassert.New(t)
+var testNamespace = "test-namespace"
 
-	presetMeshConfigMap := &corev1.ConfigMap{
-		TypeMeta: metav1.TypeMeta{
-			Kind:       "ConfigMap",
-			APIVersion: "v1",
-		},
-		ObjectMeta: metav1.ObjectMeta{
-			Name: presetMeshConfigName,
-		},
-		Data: map[string]string{
-			presetMeshConfigJSONKey: `{
+var testMeshConfig *v1alpha1.MeshConfig = &v1alpha1.MeshConfig{
+	ObjectMeta: metav1.ObjectMeta{
+		Namespace: testNamespace,
+		Name:      meshConfigName,
+	},
+	Spec: v1alpha1.MeshConfigSpec{},
+}
+
+var testPresetMeshConfigMap *corev1.ConfigMap = &corev1.ConfigMap{
+	TypeMeta: metav1.TypeMeta{
+		Kind:       "ConfigMap",
+		APIVersion: "v1",
+	},
+	ObjectMeta: metav1.ObjectMeta{
+		Name:      presetMeshConfigName,
+		Namespace: testNamespace,
+	},
+	Data: map[string]string{
+		presetMeshConfigJSONKey: `{
 "sidecar": {
-  "enablePrivilegedInitContainer": false,
-  "logLevel": "error",
-  "maxDataPlaneConnections": 0,
-  "envoyImage": "envoyproxy/envoy-alpine@sha256:6502a637c6c5fba4d03d0672d878d12da4bcc7a0d0fb3f1d506982dde0039abd",
-  "initContainerImage": "openservicemesh/init:latest-main",
-  "configResyncInterval": "2s"
+"enablePrivilegedInitContainer": false,
```

Comment:
Indentation was removed?

Reply:
didn't impact tests but I added back the indentation for readability. thanks for noticing.